server/resp/src/main/java/org/infinispan/server/resp/logging/Log.java

```diff
@@ -0,0 +1,36 @@
+package org.infinispan.server.resp.logging;
+
+import static org.jboss.logging.Logger.Level.WARN;
+
+import java.nio.CharBuffer;
+
+import org.infinispan.commons.CacheConfigurationException;
+import org.jboss.logging.BasicLogger;
+import org.jboss.logging.annotations.Cause;
+import org.jboss.logging.annotations.LogMessage;
+import org.jboss.logging.annotations.Message;
+import org.jboss.logging.annotations.MessageLogger;
+
+/**
+ * Log abstraction for the Resp protocol server module. For this module, message ids
+ * ranging from 12001 to 13000 inclusively have been reserved.
+ *
+ * @author William Burns
+ * @since 14.0
+ */
+@MessageLogger(projectCode = "ISPN")
+public interface Log extends BasicLogger {
+   @Message(value = "Cache '%s' has expiration enabled which violates the RESP protocol.", id = 12001)
+   CacheConfigurationException invalidExpiration(String cacheName);
+
+   @Message(value = "Cannot enable authentication without an authenticator.", id = 12002)
+   CacheConfigurationException authenticationWithoutAuthenticator();
+
+   @LogMessage(level = WARN)
+   @Message(value = "Encountered an unexpected exception.", id = 12003)
```

Comment:
```suggestion
   @Message(value = "Received an unexpected exception from the JVM.", id = 12003)
```

Reply:
Not sure if I like specifying JVM here. The point is we don't know what the error was exactly.

Reply:
> Not sure if I like specifying JVM here. The point is we don't know what the error was exactly.

+1

Reply:
Thanks, guys. I was thinking maybe it'd make sense if the exception was something that happened "within the JVM". If the scope was outside the handler but confined to the JVM, like an OOM exception, then it might be more meaningful to the user.

I would like to change "Encountered" to "Received" because that reduces alliteration and also fits with the UnexpectedException description in Java docs.

Reply:
I can change it to received, sure.